ProcessLib/ProcessOutput.cpp

```diff
@@ -66,6 +115,7 @@ void doProcessOutput(
 
     auto const& output_variables = process_output.output_variables;
     std::set<std::string> already_output;
+    already_output.insert("integration_point");
```

Comment:
Potentially. For now there is a process-specific output generated, so everything is dumped. Adding a choice here seems possible *e.g.* over the field numbers of the protobuf messages, which can be improved with a string-to-int map ...